src/main.ts

```diff
@@ -1,6 +1,6 @@
 import { resolve } from 'path';
 import * as core from '@actions/core';
-import { LocalProgramArgs, LocalWorkspace } from '@pulumi/pulumi/automation';
+import {LocalProgramArgs, LocalWorkspace, LocalWorkspaceOptions} from '@pulumi/pulumi/automation';
```

Comment:
nitpicking: Can you run prettier? Looks like there are some styling stuff missing :)